### PR TITLE
[zh] improve `egress` translation for egress docs

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Make translation of `egress` follow standard:
The `Egress Gateway` translate to `Egress 网关`
The `egress traffic` translate to `出口流量`

And with some translation corrections.

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/egress-control/index.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Istio 代理允许调用未知的服务。如果这个选项设置为 `REGISTRY_
     HTTP/2 200
     {{< /text >}}
 
-恭喜！您已经成功地从网格中发送了 Egress 流量。
+恭喜！您已经成功地从网格中发送了出口流量。
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: 为啥这里 Egress 要翻，而 `Egress 网关`中的不翻呢，有什么说法吗

Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: 主要是参考了大部分文档的翻译决定这样统一一下。

Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: 这样就有了不一致的问题，特别是对于新手而言，译与不译是个问题

Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: 是的，我对比了整个 egress 下的所有内容，发现大部分是按这样的方式翻译的，目前的修改算是影响最小的，新手翻译的话只能咱们 review 的时候提醒一下，如果能有一个想 K8s 社区那样的指导页其实是最好的方式。

Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: It's time to add a zh-localization guide like [k8s zh style guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#special-words). 

We can raise an issue to discuss more.

Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: Yes, lot of standards can be specific in a guide.

Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/egress-control/index.md:94]: Done: https://github.com/istio/istio.io/issues/15053

**Review by windsonsea — APPROVED** [review]
some nits can be improved later

**Review by zirain — APPROVED** [review]

**Pull request merged**